pkg/columns/group/group_test.go

```diff
@@ -15,9 +15,11 @@
 package group
 
 import (
-	"reflect"
 	"testing"
 
+	"github.com/stretchr/testify/require"
+	"gotest.tools/v3/assert"
```

Comment:
This is probably on me, sorry - should use "github.com/stretchr/testify/assert" instead.